apple/RNGHStylusData.h

```diff
@@ -0,0 +1,82 @@
+//
+//  RNGHStylusData.h
+//  Pods
+//
+//  Created by Michał Bert on 18/09/2024.
+//
+
+#ifndef RNGHStylusData_h
+#define RNGHStylusData_h
+
+@interface StylusData : NSObject
+
+@property (atomic, assign) double tiltX;
+@property (atomic, assign) double tiltY;
+@property (atomic, assign) double altitudeAngle;
+@property (atomic, assign) double azimuthAngle;
+@property (atomic, assign) double pressure;
+
+- (NSDictionary *)toDictionary;
+
+@end
+
+typedef struct {
+  double tiltX;
+  double tiltY;
+} Tilt;
```

Comment:
Same

Reply:
Done in [3b1ac10](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/3b1ac10e458a92ffbb3cbafdbc9024c36f122074)